firmware/controllers/core/error_handling.cpp

```diff
@@ -29,6 +29,8 @@ const char* getCriticalErrorMessage(void) {
 void checkLastBootError() {
 #if EFI_BACKUP_SRAM
 	auto sramState = getBackupSram();
+	if (sramState == NULL)
```

Comment:
why EFI_BACKUP_SRAM if on backup RAM? i kind of what to do in the opposite direction remove null check in other places